package.json

```diff
@@ -25,7 +28,8 @@
         "react-dom": "^16.13.1",
         "react-json-tree": "^0.11.2",
         "react-monaco-editor": "^0.36.0",
-        "typescript-to-lua": "^0.32.2"
+        "typescript-to-lua": "^0.32.2",
+        "zlib": "^1.0.5"
```

Comment:
That's a builtin dependency for node, and webpack comes with a browser version too (which is `browserify-zlib`, based on `pako`)